### PR TITLE
Br chmod command

### PR DESCRIPTION
**Pull request by boazfrider**
Here is the implementation of chmod command. 
Note : it seems to be work manually, I didn't try to test with Gtest.

### CODE REVIEW

File: BuiltInCommands.cpp [hunk 1]
```diff
@@ -228,4 +228,25 @@ void KillCommand::execute()
...
-#pragma endregion KillCommand
\ No newline at end of file
+#pragma endregion KillCommand
+
+#pragma region ChangeFileModeCommand
+void  ChangeFileModeCommand::execute()
+{
+    if (cmd_v.size() != 3)
+    {
+       perror("smash error: chmod: invalid arguments");
```
Comment (DanielBelokon) [inline BuiltInCommands.cpp:238]: Should use the Command::printError function instead, perror is when a syscall fails and errno is populated

File: Commands.cpp [hunk 2]
```diff
@@ -96,7 +96,7 @@ void GetFileInfoCommand::execute()
 {
     if(cmd_v.size() != 2)
     {
-        perror("smash error: gettype: invalid arguments");
+        perror("smash error: getfiletype: invalid arguments");
```
Comment (DanielBelokon) [inline Commands.cpp:99]: Command::printError

Comment (boazfrider) [reply Commands.cpp]: done

File: BuiltInCommands.cpp (hunk 1 above)
Comment (boazfrider) [reply BuiltInCommands.cpp]: done

**Review by DanielBelokon — APPROVED** [review]

**Pull request merged**